ts/@live-compositor/node/README.md

```diff
@@ -30,19 +30,36 @@ async function run() {
     ip: '127.0.0.1',
     transportProtocol: 'udp',
     video: {
-      encoder: {
-        type: 'ffmpeg_h264',
-        preset: 'ultrafast',
-      },
-      resolution: {
-        width: 1920,
-        height: 1080,
-      },
+      encoder: { type: 'ffmpeg_h264', preset: 'ultrafast' },
+      resolution: { width: 1920, height: 1080 },
       root: <ExampleApp />,
     },
+    audio: {
+      encoder: {
+        type: 'opus',
+        channels: 'stereo',
+      },
+    }
   });
 
   await compositor.start();
 }
 run();
 ```
+
+## Documentation
+
+API is documented via TypeScript types definitions in this package. You can also refer to [LiveCompositor server documentation](https://compositor.live/docs/api/routes), most of the APIs there translate one-to-one to methods on `LiveCompositor` instance.
+
+See https://compositor.live/docs
+
+## License
+
+`@live-compositor/node` is MIT licensed, but internally it is downloading and using Live Compositor server that is licensed
+under [Business Source License 1.1](https://github.com/software-mansion/live-compositor/blob/master/LICENSE).
+
+## LiveCompositor is created by Software Mansion
+
+[![swm](https://logo.swmansion.com/logo?color=white&variant=desktop&width=150&tag=live-compositor-github 'Software Mansion')](https://swmansion.com)
+
+Since 2012 [Software Mansion](https://swmansion.com) is a software agency with experience in building web and mobile apps. We are Core React Native Contributors and experts in dealing with all kinds of React Native issues. We can help you build your next dream product – [Hire us](https://swmansion.com/contact/projects?utm_source=live-compositor&utm_medium=readme).
```

Comment:
Maybe it would be worth mentioning something related to multimedia

Reply:
+1